docs/src/quickstart.md

```diff
@@ -3,20 +3,29 @@
 This page introduces the first steps to set up `ExaPF.jl`.
 We show how to load a power network instance and how to solve
 the power flow equations both on the CPU and on the GPU.
+The full script is implemented in [this file](https://github.com/exanauts/ExaPF.jl/tree/master/test/quickstart.jl)
 
-### How to load a MATPOWER instance?
-We start by importing into `ExaPF` an instance specified in the MATPOWER format.
+We start by importing CUDA and KernelAbstractions:
+```julia-repl
+julia> using CUDA
```

Comment:
I think it would be better to remote `julia>` as this allows a user to copy paste the commands, like here in [CUDA.jl](https://juliagpu.github.io/CUDA.jl/stable/)